src/components/sections/home/whyMun.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import { Grid, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import LocalLibraryIcon from '@material-ui/icons/LocalLibrary'
+import GroupIcon from '@material-ui/icons/Group'
+import CastForEducationIcon from '@material-ui/icons/CastForEducation'
+const useStyles = makeStyles(theme => ({
+  containerContent: {
+    color: theme.palette.primary.main,
+
+    '&:hover > div > h6, &:hover > div > p': {
+      backgroundColor: theme.palette.primary.main,
+      color: 'white',
+      transition: '1s',
+    },
+    '&:hover > div > h5': {
+      color: theme.palette.primary.main,
+      transition: '1s',
+    },
+  },
+  borderPink: {
+    border: `2px solid ${theme.palette.primary.main}`,
+  },
+}))
+
+function WhyMun() {
+  const classes = useStyles()
+  return (
+    <div
+      className='text-center'
+      style={{
+        minHeight: '60vh',
+      }}
+    >
+      <Typography
+        variant='h4'
+        style={{ color: '#000a2a' }}
+        className='pt-5'
+        color='primary'
+      >
+        JECRC MUN Excellencies
+      </Typography>
+      <img
+        src='/images/line.png'
+        className='mx-auto pb-5'
+        alt='---------------------'
+      />
+      <Grid container justify='center'>
+        <Grid item md={4}>
+          <div
+            className={`my-10 mx-10 justify-center text-center flex flex-col ${classes.containerContent}`}
+          >
+            <div className='flex justify-center'>
```

Comment:
@Rajani169 why you need these two extra divisions ??

Reply:
outer div is for the whole card and inner div is for the circlular icons